pypeit/metadata.py

```diff
@@ -1,10 +1,10 @@
 """
 Provides a class that handles the fits metadata required by PypeIt.
 """
-from __future__ import print_function
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import unicode_literals
+#from __future__ import print_function
```

Comment:
These can be removed

Reply:
Done.  And I just did all the rest of them, too...